### ISSUE TITLE
New decedents of young adult onset (HP:0011462)

### ISSUE THREAD

**Issue by jsaliba10** [obophenotype/human-phenotype-ontology#7153]
Our group of clinicians, lab directors, researchers, and biocurators from the [ClinGen Pediatric Cancer Taskforce](https://www.clinicalgenome.org/working-groups/somatic/pediatric-task-team/), [CIViC](civicdb.org), and [Disease Ontology](https://disease-ontology.org) is looking to improve and highlight pediatric and young adult cancer variants through enhanced curation protocols. 

Young Adult onset (HP:0011462) has a wide age range: between 16-40 years old. Similar to pediatric onset (HP:0410280) with 3 descendent terms, we propose adding subclasses under young adult onset. This segmentation will improve curation efforts and provide better representation and tracking of variants within the young adult population and cancer types.

Proposed age ranges were based on the information in [26812562](https://www.liebertpub.com/doi/10.1089/jayao.2011.1505), pediatric clinical trials capping their ages at 18 year-olds, and ages included in clinical treatment protocols of various cancer types. 

**Preferred term label:**
Early young adult onset
Intermediate young adult onset
Late young adult onset

**Synonyms**
N/A

**Definition (free text, please give PubMed ID)**

Early young adult onset: Onset of disease at the age >16 to under 19 [16, 19)
Intermediate young adult onset: Onset of disease at the age > 19 to under 25 [19, 25)
Late young adult onset: Onset of disease at the age > 25 to under 40 [25, 40)

**Parent term (use hpo.jax.org/app)**

https://hpo.jax.org/app/browse/term/HP:0011462

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
https://orcid.org/0000-0003-3035-2296

**Comment by azankl** [comment]
Since the exact age cut-offs are a bit arbitrary, I would include the age range into the label, e.g.: Early young adult onset (>16 to <19 years of age).
Also, there is some ambiguity in the proposed age ranges by @jsaliba10, early young adult goes to <19 and intermediate young adult start at >19, so where does a 19 year old belong?

**Comment by jsaliba10** [comment]
There are a numerous groups and entities that define onset ages differently. Our proposal is not to completely re-do the onset ranges currently used by the HPO, but to refine what is existing. When ages for the clinical trials are given they are not often specific, so it is difficult to determine if they set their cut off under 18, at exactly 18 or under 19. 

@BimalChaudhari @azankl  A mistake was made when I pasted my text as the > symbol lost its underline to designate greater than or equal to. 

In text form, the corrected age ranges proposed would  be:

Early young adult onset: Onset of disease at the age greater than or equal to 16 to under 19 [16, 19)
Intermediate young adult onset: Onset of disease at the age greater than or equal to 19 to under 25 [19, 25)
Late young adult onset: Onset of disease at the age greater than or equal to 25 to under 40 [25, 40)

This way, cases under 19 would be fall into the Early onset group, those exactly age 19 would fall into the intermediate group , and finally those age 25 would fall into the late group.

**Comment by jsaliba10** [comment]
@pnrobinson Have you and the HPO given any more thought to these term requests?

**Comment by kkrysiak** [comment]
+1

**Comment by pnrobinson** [comment]
Sorry for the delay. I am adding these terms as suggested as there were no further suggestion on the tracker.
I am adding phrases like this to the comments.
This term includes the full 16th year of age up to the completed 18th year og age (i.e., less than the 19th birthday).
The terms will be available with the February release of HPO.
